### PR TITLE
YALB 862: Add Paragraph Icons

### PR DESCRIPTION
**Pull request by ShaneMcPartlinYale**
### Description of work
- Added icons in ticket for paragraph types: gallery, media_grid, tabs, video

### Functional testing steps:
- [x] Launch the application locally
- [x] In admin interface, navigate Structure>Paragraph Types
- [x] Confirm that the pictured and highlighted icons are present for the gallery, media_grid, tabs, and video paragraph types:
image
image

### CODE REVIEW

File: web/profiles/custom/yalesites_profile/config/sync/paragraphs.paragraphs_type.media_grid_item.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ status: true
...
...TUuNS00LjktOS4yLTQuOXMtNy4xLDEuOC05LjEsNC45TDExNywxMjZsLTE1LjMtMjNjLTIuMS0yLjktNS4zLTQuNS04LjgtNC41CglzLTYuOCwxLjctOC45LDQuNWwtNDkuMSw4OC44Ii8+CjxjaXJjbGUgY3g9IjY3IiBjeT0iMzcuOCIgcj0iMzYuOCIvPgo8cmVjdCB4PSIyNzkuMiIgeT0iMC41IiB3aWR0aD0iMTcuNiIgaGVpZ2h0PSI1MTEuMSIvPgo8cmVjdCB5PSIyNDcuMiIgd2lkdGg9IjU3NiIgaGVpZ2h0PSIxNy42Ii8+CjxwYXRoIGQ9Ik01NTIuNywxOTEuOEw0ODguOCw0OC41Yy0yLjEtMy01LjUtNC45LTkuMi00LjlzLTcuMSwxLjgtOS4xLDQuOWwtNDMsNzcuNWwtMTUuMy0yM2MtMi4xLTIuOS01LjMtNC41LTguOC00LjUKCXMtNi44LDEuNy04LjksNC41bC00OS4xLDg4LjgiLz4KPGNpcmNsZSBjeD0iMzc3LjUiIGN5PSIzNy44IiByPSIzNi44Ii8+CjxwYXRoIGQ9Ik0yNDIuMiw1MDEuMWwtNjMuOS0xNDMuM2MtMi4xLTMtNS41LTQuOS05LjItNC45cy03LjEsMS44LTkuMSw0LjlsLTQzLDc3LjVsLTE1LjMtMjNjLTIuMS0yLjktNS4zLTQuNS04LjgtNC41CglzLTYuOCwxLjctOC45LDQuNWwtNDkuMSw4OC44Ii8+CjxlbGxpcHNlIHRyYW5zZm9ybT0ibWF0cml4KDAuMDMzMyAtMC45OTk0IDAuOTk5NCAwLjAzMzMgLTI4Mi4xMzc0IDQwMi41MTIxKSIgY3g9IjY3IiBjeT0iMzQ3LjEiIHJ4PSIzNi44IiByeT0iMzYuOCIvPgo8cGF0aCBkPSJNNTUyLjcsNTAxLjFsLTYzLjktMTQzLjNjLTIuMS0zLTUuNS00LjktOS4yLTQuOXMtNy4xLDEuOC05LjEsNC45bC00Myw3Ny41bC0xNS4zLTIzYy0yLjEtMi45LTUuMy00LjUtOC44LTQuNQoJcy02LjgsMS43LTguOSw0LjVsLTQ5LjEsODguOCIvPgo8Y2lyY2xlIGN4PSIzNzcuNSIgY3k9IjM0Ny4xIiByPSIzNi44Ii8+Cjwvc3ZnPgo='
```
Comment (nJim) [inline web/profiles/custom/yalesites_profile/config/sync/paragraphs.paragraphs_type.media_grid_item.yml:8]: @ShaneMcPartlinYale This one can be reverted back to the `null` value. A media_grid_item will never appear in the paragraph picker interface, so it does not need an icon. This "item" refers to a single image so the icon added in this PR kinda misrepresents its purpose as the icon shows four images.

**Review by nJim — CHANGES_REQUESTED** [review]
You got the right idea. I know this PR was mostly about setting up your localdev, navigating Drupal, and learning how to contribute to the project. Three of these icons are correct, but I have two changes to request:

1. The media_grid_item does not need an icon. Please change this back to null. This is commented below.
2. The 'tabs' paragraph needs an icon but it is missing from the PR. Please check the JIRA ticket for the SVG.

File: web/profiles/custom/yalesites_profile/config/sync/paragraphs.paragraphs_type.media_grid_item.yml (hunk 1 above)
Comment (ShaneMcPartlinYale) [reply web/profiles/custom/yalesites_profile/config/sync/paragraphs.paragraphs_type.media_grid_item.yml:8]: My mistake, I must have gotten lost in the mix for this one. Will address promptly.

**Review by nJim — APPROVED** [review]
Solid. Everything displays correctly. Ship it.

**Pull request merged**